_docs/reference/os_support.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: guide
+title: StorageOS Docs - OS support
+anchor: reference
+module: reference/ossupport
+---
+
+# OS support
+
+StorageOS uses FUSE known as a Filesystem in Userspace. It is a software interface for Unix-like computer operating systems that lets non-privileged users create their own file systems without editing kernel code. In addition, StorageOS is developed using Linux-IO (LIO) Target. An open-source implementation of the SCSI target.
+However, LIO is supported by most of the kernels available nowadays, some distributions have left the kernel module out of the main kernel package.
+
+
+## Distribution support
+
+The following table shows which distributions can enable the kernel modules to use LIO, whether needed to install linux-virtual-extra or not.
```

Comment:
"whether needed to install linux-virtual-extra or not." => "whether extra packages are required".

Reply:
Thank you! Done https://github.com/storageos/storageos.github.io/pull/148/commits/85f8b90c9002e67efe6eff881dff4b73ed98fad3